src/Symfony/Component/Workflow/SupportStrategy/InstanceOfSupportStrategy.php

```diff
@@ -31,6 +32,10 @@ public function __construct(string $className)
      */
     public function supports(WorkflowInterface $workflow, object $subject): bool
     {
+        if ($subject instanceof \stdClass && property_exists($subject, 'class')) {
```

Comment:
Is it really worth it?

Reply:
consistency is part of a good programming. Unfortunately, it is not PHP's strong suit.

I'm not a big fan of this.

Might not be worth it, indeed.